internal/web/templates/components/pkg-detail-header.html

```diff
@@ -114,7 +114,26 @@ <h1 class="text-reset m-0">
             Auto-Update:
             <strong>{{ .AutoUpdate }}</strong>
           </span>
+          <span
+            class="badge p-1 fw-normal {{ if eq .Status.Status " Failed" }}
```

Comment:
`" Failed"` should be `"Failed"`, but to be honest now that I have tried it out, I don't know about this whole `if/else`.
I think the failed status could be shown the same as every other status without any special formatting? Because there is already the flashy red "Installation Failed" on the top right + the alert box below to show the message. But I'm not sure.

Reply:
If you want to keep it that way, I suggest to change the template code a bit, since right now it's confusing to read but also seems to mess with IDEs (at least mine). As an alternative, you define make two little subtemplates in the same file, one for regular status badge, one for failed status badge. Then it would be a bit easier to read, then the `if / else` templating inside the `class` attribute.